src/_griffe/expressions.py

```diff
@@ -25,15 +24,84 @@
 
     from _griffe.models import Class, Module
 
+# https://docs.python.org/3/reference/expressions.html#operator-precedence
+_PRECEDENCE = {
+    "ExprName": 18,
+    "ExprConstant": 18,
+    "ExprList": 18,
+    "ExprTuple": 18,
+    "ExprSet": 18,
+    "ExprDict": 18,
+    "ExprAttribute": 17,
+    "ExprSubscript": 17,
+    "ExprCall": 17,
+    "ExprUnaryOp": {"~": 12, "+": 12, "-": 12, "not": 6},
+    "ExprBinOp": {
+        "**": 13,
+        "*": 11,
+        "@": 11,
+        "/": 11,
+        "//": 11,
+        "%": 11,
+        "+": 10,
+        "-": 10,
+        "<<": 9,
+        ">>": 9,
+        "&": 8,
+        "^": 7,
+        "|": 6,
+    },
+    "ExprCompare": 7,
+    "ExprBoolOp": {"and": 5, "or": 4},
+    "ExprIfExp": 3,
+    "ExprLambda": 2,
+    "ExprGeneratorExp": 2,
+    "ExprNamedExpr": 1,
+}
```

Comment:
I wonder if we shouldn't space them a bit more, to allow future edits if/when new language constructs are added.